benches/nosqlbench/tnt_hash.lua

```diff
@@ -0,0 +1,30 @@
+local console = require('console')
+console.listen("/tmp/tarantool-server.sock")
+
+box.cfg {
+    pid_file   = "./tarantool-server.pid",
+    log        = "./tarantool-server.log",
+    listen = 3301,
+    memtx_memory = 2000000000,
+    background = true,
+    checkpoint_interval = 0,
+    wal_mode = 'none',
+}
+
+s = box.schema.space.create('tester_nosqlbench')
+s:create_index('primary', {type = 'hash', parts = {1, 'unsigned'}})
+
+function try(f, catch_f)
+    local status, exception = pcall(f)
+    if not status then
+        catch_f(exception)
+    end
+end
+
+try(function()
+    box.schema.user.grant('guest', 'create,read,write,execute', 'universe')
+end, function(e)
+    print(e)
+end)
```

Comment:
In this case it is the equivalent of `xpcall()`. However the code looks strance for me. It looks like you tried to correctly load tarantool on top of existing snapshots / xlogs, but it would anyway fail on a space creation above.

If we need something like this, I would use `box.once()`.

Reply:
The same answer as [here](https://github.com/tarantool/bench-run/pull/14#discussion_r546418427)